### ISSUE TITLE
use `isolated-vm` instead of `vm`?

### ISSUE THREAD

**Issue by mishushakov** [vercel/edge-runtime#218]
## Questions & Answers

**Context of your question**

I feel like `vm` module is not secure enough to allow running arbitrary code and would like to ask, why didn't you instead opt-in for v8 isolates like Deno did?

**Comment by Kikobeats** [comment]
Yes, we're exploring moving on the implementation to use isolated-vm instead 🙂
